neurology.csl

```diff
@@ -11,13 +11,15 @@
     <category field="medicine"/>
     <issn>0028-3878</issn>
     <eissn>1526-632X</eissn>
-    <updated>2013-11-20T16:58:38+00:00</updated>
+    <updated>2015-01-31T19:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="en">
     <terms>
-      <term name="retrieved">available</term>
+      <term name="retrieved">accessed</term>
+      <term name="from">at</term>
       <term name="section" form="short">sect.</term>
+      <term name="internet">online</term>
     </terms>
```

Comment:
at, accessed, and online all have terms of their own. Use those rather than redifining other ones, please. This is too confusing.